src/components/vamcSystemPoliciesPage/query.test.ts

```diff
@@ -68,4 +87,41 @@ describe('VamcSystemPoliciesPage formatter', () => {
     expect(result.menu).toHaveProperty('data')
     expect(result).toHaveProperty('title', 'Policies')
   })
+
+  test('formats data without Lovell context', () => {
+    const mockDataStructure: VamcSystemPoliciesPageData = {
+      entity: mockData,
+      menu: mockMenu,
+      lovell: undefined,
+    }
+
+    const result = formatter(mockDataStructure)
+
+    expect(result.lovellVariant).toBeUndefined()
+    expect(result.lovellSwitchPath).toBeNull()
+  })
+
+  test('should set the lovellSwitchPath correctly for Lovell VA variant', () => {
+    const result = formatter({
+      entity: {
+        ...mockData,
+        // Override the path to simulate a Lovell VA path
+        path: {
+          ...mockData.path,
+          alias: '/lovell-federal-health-care-va/policies',
+        },
+      },
+      menu: mockMenu,
+      lovell: {
+        variant: 'va',
+        isLovellVariantPage: true,
+      },
+    })
+
+    expect(result.lovellSwitchPath).toBe(
+      '/lovell-federal-health-care-tricare/policies'
+    )
+  })
+
+  // NOTE: There is no TRICARE variant of this content type
```

Comment:
I'm pretty sure that's true, anyhow. CC @laflannery 

The `formatter()` would work just fine if it _did_ exist, though.

Reply:
Yes - no TRICARE variant, that is built with the VAMC Detail page. That is because that the CC content doesn't apply to TRICARE so it would have been too much customization.